docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
+services:
+  web:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile
+      # args:
+      #   RAILS_VER: ${RUBY_VER}
+      #   GIT_URL: ${GIT_URL}
+    ports:
+      - 80:80
+    environment:
+      APP_DB_DATABASE: ${POSTGRES_DB}
```

Comment:
@AdrianTeh 
These aren't needed - docker-compose will load env values contained inside the `.env` automatically.
See [https://docs.docker.com/compose/env-file](https://docs.docker.com/compose/env-file) for more information.